DemoBoolean.java

```diff
@@ -0,0 +1,11 @@
+package com.edocti.jintro.lab01;
+	
+public class DemoBoolean{
+	public static void main(String[] args){
+		boolean isOld=(age > 65);
+		int age = Integer.parseInt(args[0]);
```

Comment:
I don't think your code compiles. `age` is defined after `isOld`, but `isOld` depends on `age` => `isOld` must be defined after `age`.